Analysis/Tasks/eventSelection.cxx

```diff
@@ -81,33 +81,40 @@ struct EventSelectionTask {
     ccdb->setLocalObjectValidityChecking();
   }
 
-  void process(aod::Run2MatchedSparse::iterator const& collision, aod::BCsWithTimestamps const&, aod::Zdcs const& zdcs, aod::FV0As const& fv0as, aod::FV0Cs const& fv0cs, aod::FT0s const& ft0s, aod::FDDs const& fdds)
+  void process(
+    aod::Run2MatchedSparse::iterator const& col,
+    soa::Join<aod::BCsWithTimestamps, aod::Run2BCInfos> const& bcs,
```

Comment:
Could you add a using statement for the join, so that it is less error prone with line 93?